documentation/modules/deploying/proc-deploy-topic-operator-with-cluster-operator.adoc

```diff
@@ -7,6 +7,12 @@
 
 [role="_abstract"]
 This procedure describes how to deploy the Topic Operator using the Cluster Operator.
+The Topic Operator can be deployed for use in either bidirectional mode or unidirectional mode.
+In unidirectional mode, topic management is exclusively handled through `KafkaTopic` resources. 
```

Comment:
I think this is a bit confusing. The Topic Operator manages it only through KafkaTopic resources. But users can manage some topics using the Kafka Admin API as well. So I think I would remove this sentence.